pkg/sku/aws_sku_handler.go

```diff
@@ -13,55 +13,55 @@ func NewAwsSKUHandler() CloudSKUHandler {
 		{SKU: "p3.8xlarge", GPUCount: 4, GPUMemGB: 64, GPUModel: "NVIDIA V100"},
 		{SKU: "p3.16xlarge", GPUCount: 8, GPUMemGB: 128, GPUModel: "NVIDIA V100"},
 		{SKU: "p3dn.24xlarge", GPUCount: 8, GPUMemGB: 256, GPUModel: "NVIDIA V100"},
-		{SKU: "p4d.24xlarge", GPUCount: 8, GPUMemGB: 320, GPUModel: "NVIDIA A100"},
-		{SKU: "p4de.24xlarge", GPUCount: 8, GPUMemGB: 640, GPUModel: "NVIDIA A100"},
-		{SKU: "p5.48xlarge", GPUCount: 8, GPUMemGB: 640, GPUModel: "NVIDIA H100"},
-		{SKU: "p5e.48xlarge", GPUCount: 8, GPUMemGB: 1128, GPUModel: "NVIDIA H200"},
-		{SKU: "p5en.48xlarge", GPUCount: 8, GPUMemGB: 1128, GPUModel: "NVIDIA H200"},
-		{SKU: "g6.xlarge", GPUCount: 1, GPUMemGB: 24, GPUModel: "NVIDIA L4"},
-		{SKU: "g6.2xlarge", GPUCount: 1, GPUMemGB: 24, GPUModel: "NVIDIA L4"},
-		{SKU: "g6.4xlarge", GPUCount: 1, GPUMemGB: 24, GPUModel: "NVIDIA L4"},
-		{SKU: "g6.8xlarge", GPUCount: 1, GPUMemGB: 24, GPUModel: "NVIDIA L4"},
-		{SKU: "g6.16xlarge", GPUCount: 1, GPUMemGB: 24, GPUModel: "NVIDIA L4"},
-		{SKU: "gr6.4xlarge", GPUCount: 1, GPUMemGB: 24, GPUModel: "NVIDIA L4"},
-		{SKU: "gr6.8xlarge", GPUCount: 1, GPUMemGB: 24, GPUModel: "NVIDIA L4"},
-		{SKU: "g6.12xlarge", GPUCount: 4, GPUMemGB: 96, GPUModel: "NVIDIA L4"},
-		{SKU: "g6.24xlarge", GPUCount: 4, GPUMemGB: 96, GPUModel: "NVIDIA L4"},
-		{SKU: "g6.48xlarge", GPUCount: 8, GPUMemGB: 192, GPUModel: "NVIDIA L4"},
+		{SKU: "p4d.24xlarge", GPUCount: 8, GPUMemGB: 320, GPUModel: "NVIDIA A100", NVMeDiskEnabled: true},
```

Comment:
Do we know if the CSI driver works on non-AKS clusters?

Reply:
Yes. It works. no vendor locked in.